system-validation-test/features/ingest_basic.feature

```diff
@@ -4,50 +4,58 @@ Feature: Basic Ingest API calls
   Background:
     Given I use the authorization token from environment variable "CMR_TOKEN"
     And the provider from environment variable "CMR_TEST_PROV" exists
-    And set body to the following XML "<Collection><ShortName>TestColl1</ShortName><VersionId>Version01</VersionId><InsertTime>1999-12-31T19:00:00-05:00</InsertTime><LastUpdate>1999-12-31T19:00:00-05:00</LastUpdate><LongName>CMR8021COLLECTION</LongName><DataSetId>LarcDatasetId</DataSetId><Description>A minimal valid collection</Description><Orderable>true</Orderable><Visible>true</Visible></Collection>"
+
+    And set body to the following XML "<Collection><ShortName>TestCollection8021</ShortName><VersionId>Version01</VersionId><InsertTime>1999-12-31T19:00:00-05:00</InsertTime><LastUpdate>1999-12-31T19:00:00-05:00</LastUpdate><LongName>CMR8021COLLECTION</LongName><DataSetId>LarcDatasetId01</DataSetId><Description>A minimal valid collection</Description><Orderable>true</Orderable><Visible>true</Visible></Collection>"
 
   @ingest
   Scenario: Ingest of a Collection
     Given I am ingesting a "Collection"
-    And I add extension "providers/PROV1/collections/TestColl1"
+    And I add extension "providers/DEMO_PROV/collections/TestCollection8021"
     And I add header "Content-type=application/echo10+xml"
     When I submit a "PUT" request
     Then the response status code is in "200,201"
 
   @ingest
   Scenario: Searching for ingested Collection
     Given I am ingesting a "Collection"
-    And I add extension "providers/PROV1/collections/TestColl1"
+    And I add extension "providers/DEMO_PROV/collections/TestCollection8021"
```

Comment:
Could we make this a bit more precise wording wise? Like a catcher for `And I use path ...` since extension makes me thing of something like `.json` or `.xml`

Reply:
That's a good point -- I made a new handler specifying adding/setting a url path for better clarity and updated the resource_url constructor to concat the extension and url path in the proper order.